internal/inspect/table_sizes/table_sizes.go

```diff
@@ -18,7 +18,7 @@ SELECT c.relname AS name,
   pg_size_pretty(pg_table_size(c.oid)) AS size
 FROM pg_class c
 LEFT JOIN pg_namespace n ON (n.oid = c.relnamespace)
-WHERE n.nspname NOT IN ('pg_catalog', 'information_schema')
+WHERE n.nspname NOT IN ('pg_catalog', 'information_schema', 'auth', 'realtime', 'pgsodium', 'supabase_migrations', 'vault', 'storage')
```

Comment:
We missed out this one too.

Reply:
that was probably my poor rebasing! fixed now!

Reply:
see that you fixed this already, removed my commit